crates/core/src/rpc/minimal.rs

```diff
@@ -739,9 +739,29 @@ impl Minimal for SurfpoolMinimalRpc {
     //       it can be removed from rpc_minimal
     fn get_vote_accounts(
         &self,
-        _meta: Self::Metadata,
-        _config: Option<RpcGetVoteAccountsConfig>,
+        meta: Self::Metadata,
+        config: Option<RpcGetVoteAccountsConfig>,
     ) -> Result<RpcVoteAccountStatus> {
+        // validate inputs if provided
+        if let Some(config) = config {
+            // validate vote_pubkey if provided
+            if let Some(vote_pubkey_str) = config.vote_pubkey {
+                verify_pubkey(&vote_pubkey_str).map_err(|e| {
+                    jsonrpc_core::Error::invalid_params(format!("Invalid vote pubkey: {}", e))
+                })?;
```

Comment:
```suggestion
                verify_pubkey(&vote_pubkey_str)?;
```
verify pubkey should already be returning an error that should work. If you get a complier error here, you may just need
`verify_pubkey(&vote_pubkey_str).map_err(Into::into)?`